forwarder.go

```diff
@@ -10,24 +10,44 @@ import (
 	"net/http"
 	"os"
 	"strings"
+	"sync"
 )
 
+const workers = 8
+
 func main() {
 	log.Println("Splunk forwarder: Started")
 	defer log.Println("Splunk forwarder: Stopped")
 
+	forSplunk := make(chan string)
```

Comment:
Hi Luca,
I changed the name from forSplunk to logChan, like it more than logStream.
Here is commit https://github.com/Financial-Times/coco-splunk-http-forwarder/pull/6/commits/10227b872b6e4eee814dc81232c20ff53effdea5.
I tested the code after the change and functions as well as before. 
Are you happy with this?